mws/resource_mws_credentials_test.go

```diff
@@ -56,7 +56,7 @@ func TestResourceCredentialsCreate(t *testing.T) {
 	})
 }
 
-func TestAccountIdOnlyInState(t *testing.T) {
+func TestCredentialsAccountIdOnlyInState(t *testing.T) {
```

Comment:
Unrelated, but since the name starts with `TestAcc` it runs on the integration tests an fails due to the fixture.